api/core/workflow/nodes/iteration/entities.py

```diff
@@ -1,10 +1,24 @@
+from enum import Enum
 from typing import Any, Optional
 
 from pydantic import Field
 
 from core.workflow.nodes.base import BaseIterationNodeData, BaseIterationState, BaseNodeData
 
 
+class ErrorHandleMode(Enum):
```

Comment:
I think this enum should inherit from `str`.
```suggestion
class ErrorHandleMode(str, Enum):
```